class-wordpress-test.php

```diff
@@ -46,7 +46,7 @@ public function __construct() {}
 	 */
 	public function is_age_valid( $age ) : bool {
 
-		if ( $age > 25 ) {
+		if( $age > 25 ) {
```

Comment:
:no_entry_sign: **Error**: No space before opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeOpenParenthesis*).